sessions/js-modern-syntax/back-to-basics/css/global.css

```
@@ -0,0 +1,52 @@
+@import './variables.css';
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+
+
+body {
+    margin: 0;
+    font-family: system-ui;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background: var(--primary);
+    background-image: var(--bg-gradient);
```

```suggestion
    background: var(--bg-gradient);
```
I don't understand your way. Why not like this?

---

I got the code from [gradient-generator](https://www.joshwcomeau.com/gradient-generator/). 

I try that one the first time and didn't work. Is working now (re try it right now), but I don't if it will bring us problem.